bin/test-with-test-runner

```diff
@@ -34,6 +34,12 @@ function verify_exercise() {
     local implementation_files=($(jq -r --arg d "${dir}" --arg k "${implementation_file_key}" '$d + "/" + .files[$k][]' "${dir}/.meta/config.json"))
     local stub_files=($(jq -r --arg d "${dir}" '$d + "/" + .files.solution[]' "${dir}/.meta/config.json"))
     local results_file="${output_dir}/results.json"
+    local exercise_type=$3
+    local exercise_status=($(jq -r --arg t ${exercise_type} --arg s ${slug} '.exercises[$t][] | select(.slug == $s).status' config.json))
+
+    if [[ $exercise_status == "deprecated" ]]; then
+        return
```

Comment:
Maybe print a message here indicating that we are skipping it. That saves future headaches when someone notices in the logs that not all exercises are being tested and needs to dive in to figure out why.

```suggestion
        echo "Skipping deprecated exercise: ${slug}"
        return
```

Reply:
Sure! done